docs/windowsbrush-and-theme.md

```diff
@@ -5,7 +5,7 @@ title: Using PlatformColor and Responding to Themes
 
 ## Overview
 
-Windows supports two unique native styling/theming behaviors: one being the dark and light theme changes and the other being adaptive brushes and system colors. This article will show you how to set up your app to listen to theme changes and use the Windows brushes when and where you want.
+Windows supports two unique native styling/themeing behaviors: one being the dark and light theme changes and the other being adaptive brushes and system colors. This article will show you how to set up your app to listen to theme changes and use the Windows brushes when and where you want.

```

Comment:
theming is actually the correct spelling: https://www.spellcheck.net/misspelled-words/themeing

Reply:
@rectified95 I think neither are correct :) : https://www.spellcheck.net/misspelled-words/theming

Reply:
You could say that it's jargon, not in a dictionary, but orthographically themeing is not ok: 
https://web2.uvcs.uvic.ca/courses/elc/sample/beginner/gs/gs_53.htm :D

Reply:
fair :)
```suggestion
Windows supports two unique native styling/theming behaviors: one being the dark and light theme changes and the other being adaptive brushes and system colors. This article will show you how to set up your app to listen to theme changes and use the Windows brushes when and where you want.
```